src/common/GroupAvatar.js

```diff
@@ -53,17 +56,17 @@ export default class GroupAvatar extends PureComponent<Props> {
       height: size,
       width: size,
       borderRadius: size / 8,
-      backgroundColor: colorHashFromString(names.join(', ')),
-    };
-    const textSize = {
-      fontSize: size / 3,
-      color: foregroundColorFromBackground(frameSize.backgroundColor),
+      backgroundColor: Color(colorHashFromString(names.join(', ')))
+        .lighten(0.6)
+        .hex(),
     };
 
+    const iconColor = Color(colorHashFromString(names.join(', '))).string();
+
     return (
       <Touchable onPress={onPress}>
         <View style={[styles.frame, frameSize]}>
-          <Text style={[styles.text, textSize]}>{initialsForGroupIcon(names)}</Text>
+          <FontAwesome size={38} name="group" color={iconColor} />
```

Comment:
How did you choose 38 for the size? 🙂

Let's have the value be based on `GroupAvatar`'s `props.size`. Right now we're only using `GroupAvatar` in one place, and we're passing 48 there. But later, we might want to use it in other places, at other sizes, and it should still look appropriate when we do.

How about `props.size * 0.75`? That way, vertically at least, it can't possibly be at risk of running outside the rounded corners of the background.

Reply:
Also, instead of using `FontAwesome` here, directly, let's add another item in src/common/Icons.js, and import and use that.

Reply:
I chose to use 38 because it looked appropriate to me, sure will change that as well as per your suggestions

Reply:
Chris: I checked on `props.size * 0.75`, it is looking fine 😊👍🏻